src/components/views/dialogs/LeaveSpaceDialog.tsx

```diff
@@ -131,8 +131,13 @@ interface IProps {
 }
 
 const isOnlyAdmin = (room: Room): boolean => {
-    return !room.getJoinedMembers().some(member => {
-        return member.userId !== room.client.credentials.userId && member.powerLevelNorm === 100;
+    const userId = room.client.getUserId();
+    if (room.getMember(userId).powerLevelNorm !== 100) {
+        return false; // user is not an admin
+    }
+    return room.getJoinedMembers().every(member => {
+        // return true if every other member has a lower power level (we are highest)
```

Comment:
Does it mean that in case you have a lower than admin power level, but are still the highest in the room you should be considered the room admin then?

Reply:
Yes, powerLevelNorm is normalized, so if you are the highest then you will always be 100